ceph-build/config/definitions/ceph-build.yml

```diff
@@ -9,7 +9,7 @@
       - github:
           url: https://github.com/ceph/ceph
     execution-strategy:
-      combination-filter: (ARCH=="x86_64") || (DIST == "trusty" && ARCH == "arm64")
+      combination-filter: (ARCH=="x86_64") || (DIST == "xenial" && ARCH == "arm64")
```

Comment:
Since there aren't any other arm64 distros in our lineup, I think there is no need to specify `DIST == "xenial"` here.